src/main/java/org/codeforamerica/messaging/models/MessageBatchMetrics.java

```diff
@@ -0,0 +1,12 @@
+package org.codeforamerica.messaging.models;
+
+public interface MessageBatchMetrics {
+    Integer getAcceptedEmailCount();
+    Integer getRejectedEmailCount();
+    Integer getDeliveredEmailCount();
+    Integer getUndeliveredEmailCount();
+    Integer getAcceptedSmsCount();
+    Integer getRejectedSmsCount();
+    Integer getDeliveredSmsCount();
+    Integer getUndeliveredSmsCount();
```

Comment:
Ideally these status metrics would be related to status enums, but since we haven't been able to nail those down this seems ok